elearning/going_online/create_your_course.rst

```diff
@@ -0,0 +1,195 @@
+
+The course form will open. From here you can configure your course, give him a title and start adding content.
+Complete the following fields for your new course.
+
+*  **Course title** pick a name for your course.
+*  **Tags** Create and edit tags for your course, this can help you and students find courses with keywords. You can use
+   already created tags or create them as you go.
+* **Image** upload a png file by clicking the photo logo to illustrate your course.
+
+.. image:: media/course-config.png
+   :align: center
+   :alt: course-config
+
+Course Configuration
+====================
+
+You can configure your course when you create it or always go back after and edit the configuration for each course with
+the edit button.
+Don’t forget to always **save** when making changes to a course.
+
+.. image:: media/edit-course.png
+   :align: center
+   :alt: editing your course
+
+Options
+-------
+
+Here you will find the most important configuration for your course.
+
+Course type
+~~~~~~~~~~~
+
+First you can choose the course type. This will change the course display on the website.
+
+.. image:: media/type-documentation.png
+   :align: center
+   :alt: course-config
+
+**Training** courses will also allow you to track the progress of your students while **Documentation** is mainly used
+when contents within the course don’t specially follow each other. Here is the same course with the different displays.
+
+**Documentation**
+
+.. image:: media/type-training.png
+   :align: center
+   :alt: course-config
+
+**Training**
+
+.. image:: media/type-training2.png
+   :align: center
+   :alt: course-config
+
+Course responsible
+~~~~~~~~~~~~~~~~~~
+
+For each course you have to choose a responsible. The course responsible will have all accesses regarding the course.
+He will be able to see the list of attendees, send them emails, add content...
+
+.. image:: media/course-resp.png
+   :align: center
+   :alt: course-config
+
+The course responsible can be changed by clicking the arrow next to their name at any moment.
+If you have multiple websites you can choose which website your course is going to be displayed on.
+
+Access Rights
+~~~~~~~~~~~~~
+
+Access rights will allow you to decide who can access your courses. If you choose **Public**, anyone who finds the
+course on your website will be able to enroll and access your content.
+
+.. image:: media/enroll-policy.png
+   :align: center
+   :alt: picking the enroll policy
+
+You can send invite mails directly from the dashboard of the eLearning App.
+Go to the settings button of the course you want to invite people to and click on invite.
+This will allow you to send a mail inviting people to your course.
+
+.. image:: media/invite.png
+   :align: center
+   :alt: invite students to your course
+
+When choosing the **Enroll Policy** **On Invitation** a smart **invite** button will appear on the course. This will
+allow you to send mails to students so they can enroll.
+
```

Comment:
You can also invite people to attend the course by click on the *Share* button on the course's website page.  A link to
access the course is generated, and can be shared as many time was you like.

You also have the possibility to directly share the course on **Social Networks** with our smart buttons. -> I think this paragraph is weird here, because the user does not know how to publish the course yet, meaning that he does not know how to see this.  The first of the paragraph make make sense being here, but this second, could be displayed after the of shows how to publish a course?